nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -10,7 +10,7 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
-
+from typing import Dict
```

Comment:
Could you please add a blank line before the line `from typing import Dict`?

Reply:
Thanks for the comment! Updated